src/_content/presenters/ryan-cheley.md

```diff
@@ -0,0 +1,14 @@
+---
+company: Desert Oasis Healthcare
+hidden: false
+name: Ryan Cheley
+permalink: /presenters/ryan-cheley/
+photo: ryan-cheley.jpeg
```

Comment:
when I look at Ryan's picture on my local box, I see this: 
![image](https://github.com/user-attachments/assets/b0986314-5100-4005-85e8-b3f6fb3d449f)

That doesn't match the photo I'm adding here

Reply:
oh, that's because it's his 2023 picture that was in the stub branch: https://github.com/djangocon/2024.djangocon.us/blob/stub-schedule/src/_content/presenters/ryan-cheley.jpeg

How do I forcibly regenerate thumbnails locally?